### DISCUSSION TITLE
Commit message says „Update file“ when file is created

### DISCUSSION THREAD

**Discussion by moritztim** [community/community#145520] [Mobile]
When creating a new file (say we call it "<file>"), the commit message says "Update <file>", when the web version would say "Create <file>".
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.186.0 (146974874)
Device: iPhone X (iOS 16.7.10)
</details>

**Comment by iskae** [comment]
Hi @moritztim 👋 

Thanks for reaching out ❤️ I believe this is affecting both of our clients 😅 Creating an internal issue for changing this!

Please stay tuned for our future releases!

Cheers!
